training/data_preparation/README.md

```diff
@@ -0,0 +1,26 @@
+## How to prepare your data for training
+
+Follow the instructions in this document to prepare your data for model training.
+- [Prerequisites](#prerequisites)
+- [Preparing your data](#preparing-your-data)
+
+## Prerequisites
+No special prerequisites other than the standard data ETL tools to create a `.tsv`-formatted file.
+
+## Preparing your data
+The training data must be a sole `.tsv`-formatted file and must carry the `.tsv` extension. 
+Multiple `.tsv` files in the input bucket for training will result in an error.
+
+The file has to be made up of two columns exactly, separated by the tab-delimiter (`\t`) as is typical for a `tsv` file.
```

Comment:
I think it will be more clear here to provide a table outlining the TSV schema:

| text | label |
| - | - |
| string | string |

Reply:
Together with an example:

| text | label |
| - | - |
| Exposure to violent video games causes at least a temporary increase in aggression and this exposure correlates with aggression in the real world | neg |
| video game violence is not related to serious aggressive behavior in real life | pos |
| some violent video games may actually have a prosocial effect in some contexts | pos |

Reply:
Thank you. I have added the first table for illustration, but the second one I have kept in the format it is now cause I'd like it to be as close to the realistic dataset expectations.